pkg/cloudprovider/providers/openstack/openstack_instances.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
```

Comment:
I must have missed this getting in but the name of this file is unnecessarily recursive.